src/org/zaproxy/zap/extension/ascanrulesBeta/SessionFixation.java

```diff
@@ -662,18 +655,14 @@ else if (datediffSeconds > anHourSeconds )  {
 	        			String extraInfo = Constant.messages.getString("ascanbeta.sessionfixation.alert.cookie.extrainfo", currentHtmlParameter.getName(), cookieBack1.getValue(), (cookieBack2== null?"NULL": cookieBack2.getValue()));
 	        			String attack = Constant.messages.getString("ascanbeta.sessionfixation.alert.attack", currentHtmlParameter.getType(), currentHtmlParameter.getName());
 	        			
-	        			//and figure out the risk, depending on whether it is a login page	        
-	        			int risk = Alert.RISK_LOW;
 	        			if (loginUrl) {
 	        				extraInfo += ("\n" + Constant.messages.getString("ascanbeta.sessionfixation.alert.cookie.extrainfo.loginpage"));
-	        				//login page, so higher risk
-	        				risk = Alert.RISK_MEDIUM;
 	        			} else {
-	        				//not a login page.. lower risk
-	        				risk = Alert.RISK_LOW;
+	        				//not a login page.. skip
+	        				continue;
```

Comment:
We could do this earlier, maybe before:
https://github.com/zaproxy/zap-extensions/blob/a1b99b08b64916f039c08311f19712bdc07813ba/src/org/zaproxy/zap/extension/ascanrulesBeta/SessionFixation.java#L548-L556
there's no need to test if we are going to ignore the result?